test/reviewRating.test.js

```diff
@@ -0,0 +1,308 @@
+import { describe, expect, jest, test } from "@jest/globals";
+import { roundRating } from "../server/rating.js";
+import reviewData from "../server/sendReview.js";
+
+describe("reviewData()", () => {
```

Comment:
The tests in this suite are a bit mixed. All three seem to test `reviewData()`, but the third also separately tests `roundRating()`.